pkg/command/root.go

```diff
@@ -109,3 +131,25 @@ func NewLogger(cfg *config.Config) log.Logger {
 		log.Color(cfg.Log.Color),
 	)
 }
+
+func addRuntime(app *cli.App) {
+	app.Commands = append(app.Commands, api.Commands()...)
```

Comment:
I think we should slim down the list to services we really need. Many of these services are not yet needed by us.

Reply:
Yeah we have to decide which services we want to include